wire-ios-data-model/Resources/zmessaging.xcdatamodeld/zmessaging2.112.0.xcdatamodel/contents

```diff
@@ -320,7 +320,7 @@
         <attribute name="remoteIdentifier_data" optional="YES" attributeType="Binary"/>
         <attribute name="richProfile" optional="YES" attributeType="Binary"/>
         <attribute name="serviceIdentifier" optional="YES" attributeType="String"/>
-        <attribute name="supportedProtocols" optional="YES" attributeType="Transformable"/>
+        <attribute name="supportedProtocols" optional="YES" attributeType="Transformable" valueTransformerName="ExtendedSecureUnarchiveFromData"/>
```

Comment:
Do we need to add it here in xcdatamodel `112` if I see the same changes in `113`?

Reply:
Reverted. 👍